### ISSUE TITLE
Request "Stable Releases" to update occasionally

### ISSUE THREAD

**Issue by Kumiri** [TASVideos/desmume#303]
Even if it is updated once a year, it is very good.

"Official Nightly Builds" is constantly being updated.
"Stable Releases" has stopped updating for more than four years.
Why is "Stable Releases" stopped updating? ?
--------------------------------------------------
OpenGL 3.2 can't keep up with the times, it's easy to have problems... For example: "Pokémon Diamond and Pearl", "Pokémon HeartGold and SoulSilver".

When OpenGL3.2 plays "Pokémon Diamond", when the outdoor map moves, the background has some black spots flashing.
OpenGL3.2 plays "Pokémon HeartGold" more directly... The main screen is always black.

※ I am still using the July 1, 2017 version. I just tried the 2018-11-08 version. It does not recognize the Chinese archive name (Chinese character garbled code). For example, use the 18-11-08 version to open "Pokémon Diamond钻石.nds", the default will generate a new archive, "Pok茅mon Diamond閽荤煶.dsv".
(The archive "Pokémon Diamond钻石.dsv" is already in the "Battery" folder)
--------------------------------------------------
**### The focus is coming**

Since "Stable Releases" stops updating, the third-party languages ​​on the network are all earlier versions. Occasionally there is a 0.9.12 git#**** I don't know if it is a new version, I need to download it to determine. And I don't know if OpenGL is updated or not. I don't know if there will be obvious bugs.

### As a gamer, the requirements are simple: ①there is no obvious problem with the image quality; ②the archive format supports direct saving of *.SAV; ③the archive name supports Chinese.④Non-google download(Chinese netizens)
--------------------------------------------------
to sum up:
1. Update once a year, or replace the "Git" with the version number.
2. Real-time updates, the latest 3D renderer. Improve picture quality and color experience.
3. Solve obvious problems and alleviate the problems caused by "language of non-Unicode programs".
4. Diversified download methods, provide official website to download or provide github download.
※ I dare not ask you to launch the Chinese language, but the bug fix announcement can't be less. After 2015, the official website exist in name only.

**Comment by Jules-A** [comment]
> I am still using the July 1, 2017 version. I just tried the 2018-11-08 version.

What about the latest nightlies?? Do they have that issue? There were fixes around December 2018 for non-English characters so I'm not sure why you are using that build anyways?

The newer builds don't have any issues in the Pokemon games with black dots/lines in OpenGL EXCEPT with MSAA enabled and even then it only happens in very few specific locations and is somewhat lessened with "Smooth Textures" enabled.

I'm not sure what you are saying about the official website but the latest Nightly version can be downloaded there.

As for updating the release build, I think it would probably be worth it just to cut down on the issue submissions about issues fixed years ago but that's just me. Even if it's released with an extremely vague changelog. There hasn't been many changes lately so the code has had quite a long time to stabilize and there hasn't been any reports of stability issues reported.
